_projects/2018/plasmapy/plasma-object.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Plasma class
+desc: Devlop a new Plasma class for loading and storing multiple types of plasma physics data.
+requirements:
+ - Good knowledge of Python, including object-oriented programming
+ - Familiarity with basics of plasma physics
+ - Some experience with plasma physics data formats
```

Comment:
Maybe this could be made a bit more broad too: "Some experience with scientific data formats, especially plasma physics data formats, is preferred."